src/realm-settings/NewClientPolicyForm.tsx

```diff
@@ -143,29 +145,37 @@ export default function NewClientPolicyForm() {
       "org.keycloak.services.clientpolicy.condition.ClientPolicyConditionProvider"
     ];
 
-  const formValues = form.getValues();
-
   const save = async () => {
-    const createdForm = formValues;
+    const createdForm = form.getValues();
     const createdPolicy = {
       ...createdForm,
       profiles: [],
       conditions: [],
     };
 
-    const policyNameExists = policies.find(
-      (policy) => policy.name === createdPolicy.name
-    );
+    const getAllPolicies = () => {
+      const policyNameExists = !!policies.find(
```

Comment:
You can use [`some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) to get a boolean value directly.
```suggestion
      const policyNameExists = policies.some(
```